### DISCUSSION TITLE
Code Generation plugin hangs

### DISCUSSION THREAD

**Discussion by epayum200** [Netflix/dgs-framework#706] [Q&A]
I have a graphql schema fila and I am trying to use the code generation plugin to generate a query API.  I have the plugin configured as follows:

 ```
schemaPaths = ["${projectDir}/src/main/resources/schema"]
 packageName = 'some.package.name'
 generateClient = true 
 generateDataTypes = false
 skipEntityQueries = true
 includeQueries = ["query1", query2"]
 includeMutations = [""]
 shortProjectionNames = true
 maxProjectionDepth = 2
```
When I run gradle, the process hangs here:
```
<-------------> 0% EXECUTING [22h 50m 47s]
> :generateJava
```

Are there any debugging options I can use to figure out why it's getting stuck?

**Comment by paulbakker** [comment]
We've not seen that happening, so I'm not sure. You could try running with `--info` or `--debug`, but I'm not sure what to look for, especially because it seems to hang at the very start!?

Could you try with a very simple schema, just to verify that the setup is ok?
